logstash_async/transport.py

```diff
@@ -301,13 +307,14 @@ def __init__(
             self,
             host: str,
             port: int,
+            path: str = '',
```

Comment:
I'm not sure but it might be safer to add the new argument to end of the argument list (before `**kawrgs`) to not break clients which pass `timeout` and so on as positional arguments.